python/paddle/fluid/tests/unittests/dygraph_to_static/yolov3.py

```diff
@@ -275,6 +280,53 @@ def __init__(self, ch_in, is_train=True, use_random=False):
                 self.route_blocks_2.append(route)
             self.upsample = Upsample()
 
+    def multiclass_nms(
+        self,
+        bboxes,
+        scores,
+        score_threshold,
+        nms_top_k,
+        keep_top_k,
+        nms_threshold=0.3,
+        normalized=True,
+        nms_eta=1.0,
+        background_label=0,
+        name=None,
+    ):
+        check_variable_and_dtype(
+            bboxes, 'BBoxes', ['float32', 'float64'], 'multiclass_nms'
+        )
+        check_variable_and_dtype(
+            scores, 'Scores', ['float32', 'float64'], 'multiclass_nms'
+        )
+        check_type(score_threshold, 'score_threshold', float, 'multicalss_nms')
+        check_type(nms_top_k, 'nums_top_k', int, 'multiclass_nms')
+        check_type(keep_top_k, 'keep_top_k', int, 'mutliclass_nms')
+        check_type(nms_threshold, 'nms_threshold', float, 'multiclass_nms')
+        check_type(normalized, 'normalized', bool, 'multiclass_nms')
+        check_type(nms_eta, 'nms_eta', float, 'multiclass_nms')
+        check_type(background_label, 'background_label', int, 'multiclass_nms')
+
+        helper = LayerHelper('multiclass_nms', **locals())
+        output = helper.create_variable_for_type_inference(dtype=bboxes.dtype)
+        helper.append_op(
```

Comment:
why add this with append op, will this be used only in static mode?

Reply:
this function is copied from 
![image](https://user-images.githubusercontent.com/23737287/204726175-eb587ad4-2411-4433-9064-fc3d96d97d18.png) without modification

Reply:
Ok, This is a bad usage in eager mode and this may cause performance issue or execution error.

Reply:
how about change this to use _C_ops.xxx?

Reply:
Done